src/js/resize.js

```diff
@@ -0,0 +1,32 @@
+import Split from 'split.js'
+
+const PAGE_WIDTH = 960
+
+const isSplited = () => document.getElementsByClassName('gutter').length > 0
```

Comment:
Please use snake case for all namings i.e. `isSplited` -> `is_splited`

Reply:
Really? Look at the DOM elements naming. As long as Split.js uses camel case and I use functions from that library. Changing case would be inconsistent.

Reply:
"splited" is not a word. It's `is_split`.

Reply:
Thank you Leandros. You are very helpful, as always.

Reply:
Well our code base started with camel_case so I'd prefer to keep it as it is :)